core/web/schema/type/log.graphql

```diff
@@ -0,0 +1,28 @@
+enum LogLevel {
+    DEBUG
+    INFO
+    WARN
+    ERROR
+}
+
+type LogLevelConfig {
+    keeper: LogLevel
```

Comment:
Actually can `LogLevel` be nil?

Reply:
Yes, you can send nil to via `LogLevel`, what you cannot send is a not nil value that doesn't match the ENUM.